desktop/components/search_bar/view.coffee

```diff
@@ -169,15 +169,19 @@ module.exports = class SearchBarView extends Backbone.View
   emptyItemClick: ->
     analyticsHooks.trigger 'search:empty-item:click',
       query: @query
-      label: modelNameAndIdToLabel 'user-query', 'query'
+      item_number: 1
+      item_type: 'search'
+      destination_path: "#{sd.APP_URL}/search?q=#{@query}"
     @selected = true
     location.assign "/search?q=#{@query}"
 
   selectResult: (e, model) ->
     return @emptyItemClick() unless model
     analyticsHooks.trigger 'search:item:click',
       query: @query
-      label: modelNameAndIdToLabel model.get('display_model'), model.id
+      item_number: model.collection.models.findIndex( (result) -> return result.id == model.id ) + 1
```

Comment:
Also it appears I have a real failure. I think this can be fixed by doing `model.get('collection')` vs `model.collection`. I'll try that in my next commit.